install_files/ansible-base/roles/validate/tasks/validate_ossec_info.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Validate OSSEC Admin email address.
+  assert:
+    that:
+      - "'@' in ossec_alert_email"
+      - ossec_alert_email != "ossec@ossec.test"
+    msg: >-
+      OSSEC Admin email address must be valid.
+      This address is used to receive OSSEC notifications.
+
+- name: Validate SASL username for OSSEC config.
+  assert:
+    that:
+      # Ensure not the dummy value specified in example vars file.
+      - sasl_username != "test"
+      - "'@' not in sasl_username"
+
+- name: Validate SASL password for OSSEC config.
+  assert:
+    that:
+      # Ensure not the dummy value specified in example vars file.
+      - sasl_password != "password123"
+      - sasl_password|length >= 8
+    msg: >-
+      Password for OSSEC email account must be strong.
```

Comment:
I hear that. Definitely felt gross as soon as I wrote a check for password strength, but was trying to determine that a real password had indeed been typed in. `|length > 1` didn't feel right.